tests/unauthorized/userTest.js

```diff
@@ -8,15 +8,15 @@ describe('Unauthorized: Currently testing user management,', function () {
   const {
     invalidAuthHeader,
     xmlResponseHeaders,
-    capabilitiesGETRequestInvalidAuth,
+    getCapabilitiesWithInvalidAuthInteraction,
     createOwncloud
   } = require('../pactHelper.js')
   const {
     unauthorizedXmlResponseBody,
     createProvider
   } = require('../pactHelper.js')
 
-  const addInvalidAuthInteraction = (provider, requestName, method, path, query) => {
+  const nvalidAuthInteraction = (provider, requestName, method, path, query) => {
```

Comment:
```suggestion
  const InvalidAuthInteraction = (provider, requestName, method, path, query) => {
```
I think it is supposed to be `InvalidAuthInteraction` ( both pronounce similar though :smile: )

Reply:
good catch, thanks
I've changed it